content/docs/how-we-work/open-source-contribution/drupal-contribution.md

```diff
@@ -5,9 +5,13 @@ weight: 2
 
 # Drupal Contribution
 
-## How to start Contributions
+## How to Start Contributions
 
-### [Create a Drupal.org account](https://www.drupal.org/user/register)
+### Create a Drupal.org Account
+
+To contribute to Drupal, start by creating a Drupal.org account. This account is our gateway to engaging with the community, participating in discussions, and contributing to the platform's development.
+
+Go to [Create a Drupal. org account](https://www.drupal.org/user/register) then you need to:
```

Comment:
@Yemaneberhan-Lemma I found a typo - there is a space `Drupal. org` instead of `Drupal.org`. Please check. Rest of the changes looks good.

Reply:
@axelabhay Fixed.